external/docker/Dockerfile

```diff
@@ -1,4 +1,8 @@
-FROM dedis/conode:dev
+FROM golang:1.15-buster
+RUN apt update && apt install -y procps ca-certificates && apt clean
+WORKDIR /root/
+
+EXPOSE 7770 7771
```

Comment:
It isn't in sync with what `CMD` starts, there is more openned ports.

Reply:
fixed